README.md

```diff
@@ -32,7 +32,8 @@ allow privilege escalation.  It may increase the ability of a logged
 in user to perform denial of service attacks, however.
 
 In particular, bubblewrap uses `PR_SET_NO_NEW_PRIVS` to turn off
-setuid binaries.
+setuid binaries, which is the traditional way to get out of things
+like chroots.
```

Comment:
Looks like <https://en.wikipedia.org/wiki/Chroot#Limitations> mentions this, let's link to it?  There's probably better pages but hey it's wikipedia.  We can edit that section to mention bubblewrap and user namespaces at some point =)